# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cracklib/fpic.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Of course the shared library needs PIC - imported from ROCK Linux.

  - Rene Rebe <reneÂ@exactcode.d>

diff -Nru cracklib,2.7-orig/cracklib/Makefile cracklib,2.7/cracklib/Makefile
--- cracklib,2.7-orig/cracklib/Makefile	2004-11-06 09:48:53.010325096 +0100
+++ cracklib,2.7/cracklib/Makefile	2004-11-06 09:49:08.042039928 +0100
@@ -8,7 +8,7 @@
 
 LIB=	libcrack.a
 OBJ=	fascist.o packlib.o rules.o stringlib.o
-CFLAGS= -O -I../cracklib -DIN_CRACKLIB
+CFLAGS= -O -I../cracklib -DIN_CRACKLIB -fPIC
 
 $(LIB):	$(OBJ)
 	ar rv $(LIB) $?
